### PR TITLE
Insert empty link to navbar in hope it fixes #3422

### LINKED ISSUE #3422: Weird styling in docs navbar
### Bug report

**Bug summary**

Since 3.6.1, there's a weird offset (or lack of it) for first item in the docs main navbar. See below:

ytdocs

### PR DESCRIPTION
**Pull request by Xarthisius**
## PR Summary

Since the behavior mentioned in #3422 always applies to first item in the navbar, I inserted and empty link. However, I'm open to ideas if someone knows how to do it via e.g. css...

### CODE REVIEW

**Comment by matthewturk** [comment]
I spent a handful of minutes trying to find the correct CSS incantation to correctly negate `:not(li) > ol > li:first-child > :first-child, :not(li) > ul > li:first-child > :first-child` and could not.

**Review by matthewturk — APPROVED** [review]

**Review by chummels — APPROVED** [review]
Seems reasonable to me.

File: doc/source/conf.py [hunk 1]
```diff
@@ -130,6 +130,7 @@
     bootstrap_version="3",
     bootswatch_theme="readable",
     navbar_links=[
+        ("", ""),
```
Comment (neutrinoceros) [inline doc/source/conf.py:133]:
```suggestion
        ("", ""),  # see https://github.com/yt-project/yt/pull/3423
```

**Review by neutrinoceros — COMMENTED** [review]
this seems reasonable, though I'm unsure how to check that it actually works. In any case, I'd suggest keeping a trace of why this one line was inserted, it seems really easy to loose track of.

**Comment by Xarthisius** [comment]
> this seems reasonable, though I'm unsure how to check that it actually works. 

https://tests.yt-project.org/job/yt_docs_new/1604/artifact/doc/build/html/index.html

**Review by neutrinoceros — APPROVED** [review]

**Pull request merged**